### PR TITLE
[CARBONDATA-405]Fixed Data load fail if dataframe is created with LONG datatype column

### PR DESCRIPTION
**Pull request by ravipesala**
If the dataframe schema has long datatype then carbon table creation is failing because it cannot convert long type to supported bigint type. Same is fixed in this PR

### CODE REVIEW

**Comment by ravipesala** [comment]
http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/602/

File: integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataframe/DataFrameTestCase.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.io.File
+
+import org.apache.spark.sql.{DataFrame, Row, SaveMode}
+import org.apache.spark.sql.common.util.CarbonHiveContext._
+import org.apache.spark.sql.common.util.{CarbonHiveContext, QueryTest}
+import org.scalatest.BeforeAndAfterAll
+
+/**
+ * Test Class for hadoop fs relation
```
Comment (Jay357089) [inline integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataframe/DataFrameTestCase.scala]: the comment is not proper...

Comment (ravipesala) [inline integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataframe/DataFrameTestCase.scala]: ok

File: integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataframe/DataFrameTestCase.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import org.apache.spark.sql.{DataFrame, Row, SaveMode}
+import org.apache.spark.sql.common.util.CarbonHiveContext._
+import org.apache.spark.sql.common.util.{CarbonHiveContext, QueryTest}
+import org.scalatest.BeforeAndAfterAll
+
+/**
+ * Test Class for dataframe of carbon
+ *
+ */
+class DataFrameTestCase extends QueryTest with BeforeAndAfterAll {
```
Comment (ravipesala) [inline integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataframe/DataFrameTestCase.scala]: ok

**Pull request closed without merging**